ipam/claim.go

```diff
@@ -128,6 +128,9 @@ func (c *claim) Try(alloc *Allocator) bool {
 		// but we also cannot prove otherwise, so we let it reclaim the address:
 		alloc.debugln("Re-Claimed", c.cidr, "for ID", c.ident, "having existing ID as", existingIdent)
 		c.sendResult(nil)
+	case c.ident == "weave:expose":
```

Comment:
@bboreham Is `weave:expose` even a valid identifier for this function? AFAICT this is supposed to be a container ID, so it seems like there is a bug where `weave:expose` calls this function?

Reply:
The `weave expose` command allocates an address for the local machine on the Weave network.
Rather than have a whole different path for that, we pretend we are allocating an address for a container with ID `weave:expose`.